vaadin-cdi/src/test/java/com/vaadin/cdi/DeploymentTestSuiteIT.java

```diff
@@ -9,13 +9,11 @@
 @Suite.SuiteClasses({ CDIIntegrationWithCustomDeploymentTest.class,
         CDIUIProviderTest.class, ConventionsTest.class, MultipleRootUIsTest.class,
         CDIIntegrationWithConflictingDeploymentTest.class,
-        CDIIntegrationWithConflictingUIPathTest.class,
         CDIIntegrationWithDefaultDeploymentTest.class, RootViewAtContextRootTest.class,
         MultipleAccessIsolationTest.class, ScopedInstancesTest.class,
         InjectionTest.class, ConsistentInjectionTest.class,
         QualifiedInjectionTest.class, MultipleSessionTest.class,
         ScopedProducerTest.class, CrossInjectionTest.class, ShiroTest.class,
-        InappropriateNestedServletInDeploymentTest.class,
         InappropriateDeploymentTest.class, NonPassivatingBeanTest.class,
         UIDestroyTest.class})
```

Comment:
On my repository I've deleted this IT. Not sure whether is it needed. 
There are a few tests - at least from me - not included here.